src/link.rs

```diff
@@ -0,0 +1,23 @@
+//! Functions to open links
+
+/// If not wasm, then open link in browser. If target is wasm, then link can be opened in the same tab, or in a new tab. But when link is opened in a new tab, browser may block it and ask user permission to do it.
+pub fn open(url: &str, new_tab: bool) {
```

Comment:
`new_tab` is needed because `egui` will have it in upcoming version: https://github.com/emilk/egui/issues/231.